bin/hubot.js

```diff
@@ -26,7 +27,7 @@ const options = {
   enableHttpd: process.env.HUBOT_HTTPD !== 'false',
   scripts: process.env.HUBOT_SCRIPTS || [],
   name: process.env.HUBOT_NAME || 'Hubot',
-  path: process.env.HUBOT_PATH || '.',
+  path: process.env.HUBOT_PATH || undefined,
```

Comment:
I think this is what would happen if it's not defined, so might not be necessary. But, it can be nice to be explicit.